packages/esm-form-entry-app/src/app/form-creation/form-creation.service.ts

```diff
@@ -90,6 +90,8 @@ export class FormCreationService {
   private wireDataSources(createFormParams: CreateFormParams) {
     const visitTypeUuid = this.singleSpaPropsService.getPropOrThrow('visitTypeUuid');
     const patient = this.singleSpaPropsService.getPropOrThrow('patient');
+    const patientUuid = this.singleSpaPropsService.getPropOrThrow('patientUuid');
```

Comment:
This is redundant... You should be able to get the patient UUID from the patient object already (it's the id field).